wordpress-mobile/WordPress-iOS
# Change post list search to always happen in a list view

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/11646
Depends on https://github.com/wordpress-mobile/WordPress-iOS/pull/11816

This PR changes the post list search. As follow:

* Always display results in a compact cell
* Group posts by status (Published, Pending, Trashed)

![search](https://user-images.githubusercontent.com/7040243/58721992-ab1e1600-83ac-11e9-8d0a-e3c7f8f10913.gif)


To test:
- Check if search **always** display compact cells
- The behavior of the search should remain the same
- Test in iPad and RtL screens
